<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCompanyForm</class>
 <widget class="QDialog" name="AddCompanyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>439</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Company Form</string>
  </property>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>371</width>
     <height>285</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="afmLabel">
      <property name="text">
       <string>AFM</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="afmLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="text">
       <string>Phone</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="phoneLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cnameLabel">
      <property name="text">
       <string>Company's Name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cnameLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="rNameLabel">
      <property name="text">
       <string>Representative's Name</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="rNameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="rSurnameLabel">
      <property name="text">
       <string>Representative's Surname</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLineEdit" name="rSurnameLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="discountPercentageLabel">
      <property name="text">
       <string>Discount percentage</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLineEdit" name="discountPercentageLineEdit"/>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="fAXLabel">
      <property name="text">
       <string>FAX</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QLineEdit" name="fAXLineEdit"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="usernameLabel">
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLineEdit" name="usernameLineEdit"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="passwordLabel">
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="passwordLineEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
